### PR TITLE
Disable failing tests

### PR DESCRIPTION
**Pull request by chapulina**
Having constant unstable CI makes it harder to notice when new test failures are introduced. The goal is to have green CI across the board and keep it that way.

This PR is just disabling known test failures, which have been listed on #75 . We should go back and fix these asap instead of just leaving them there forever.

### CODE REVIEW

File: src/plugins/Scene3D.cc [hunk 1]
```diff
@@ -1105,8 +1111,14 @@ void TextureNode::PrepareNode()
     delete this->texture;
     // note: include QQuickWindow::TextureHasAlphaChannel if the rendered
     // content has alpha.
+#if QT_VERSION < QT_VERSION_CHECK(5, 14, 0)
     this->texture = this->window->createTextureFromId(
         newId, sz, QQuickWindow::TextureIsOpaque);
+#else
+    // TODO(anyone) Use createTextureFromNativeObject
+    this->texture = this->window->createTextureFromId(
+        newId, sz, QQuickWindow::TextureIsOpaque);
```
Comment (chapulina) [inline src/plugins/Scene3D.cc:1120]: @iche033, homebrew is already using Qt 5.14 and there's a new deprecation warning:

`'createTextureFromId' is deprecated: Use createTextureFromNativeObject() instead`

See [Obsolete Members for QQuickWindow](https://doc.qt.io/qt-5/qquickwindow-obsolete.html) and [QQuickWindow::createTextureFromNativeObject](https://doc.qt.io/qt-5/qquickwindow.html#createTextureFromNativeObject). Do you think that fixing this warning will be quick or does it need more thought? I didn't dig too deep to find out how to get the `nativeObjectPtr`.

**Comment by j-rivero** [comment]
I've [implemented a couple of helper functions](https://github.com/ignitionrobotics/ign-gui/compare/chapuilna/failing_tests...chapuilna/failing_tests_jrivero) in `chapuilna/failing_tests_jrivero`

**Comment by chapulina** [comment]
Thanks, Jose, those look great! Feel free to push to this branch!

**Comment by chapulina** [comment]
Thanks for the fix, @j-rivero ! Your commit didn't have DCO, so I squashed them all to add DCO and you disappeared as an author :confused: Sorry :cry:

**Comment by chapulina** [comment]
@j-rivero , we're having some compilation problems on Windows:

```
D:\Jenkins\workspace\ign_gui-pr-win\ws\ign-gui\src\MainWindow_TEST.cc(50,1): error C2011: 'MainWindowTest_DISABLED_Testname_Test': 'class' type redefinition [D:\Jenkins\workspace\ign_gui-pr-win\ws\build\ignition-gui2\src\UNIT_MainWindow_TEST.vcxproj]
```

I tried something on e6837017f6994916acf7377ee7b0d52113e27914 but I think we may need some `#` operators :thinking:

**Comment by j-rivero** [comment]
I've changed this PR to use https://github.com/ignitionrobotics/ign-cmake/pull/102 that provides the filter macros we were using here before.
Tested in: [Build Status](https://build.osrfoundation.org/view/All/job/ign_gui-pr-win/330/)

**Comment by j-rivero** [comment]
This run on Windows check these changes together with ign-cmake2 support: [Build Status](https://build.osrfoundation.org/view/All/job/ign_gui-pr-win/386/) Seems to do what it is expected to do, no test failures.

**Pull request merged**